controls/imageeditor/overview.md

```diff
@@ -1,22 +1,22 @@
 ---
 title: Overview
 page_title: .NET MAUI ImageEditor Documentation - Overview
-description: Review the Telerik UI for .NET MAUI ImageEditor control.
+description: Try now the Telerik ImageEditor for .NET MAUI for editing and saving images in your mobile and desktop applications.
 position: 0
 slug: imageeditor-overview
 ---
 
 # .NET MAUI ImageEditor Overview
 
-Telerik .NET MAUI ImageEditor is a control that enables you to visualize and edit images in different file formats in your mobile and desktop application. As an addition to the ImageEditor control you can use the predefined UI with all built-in commands - RadImageEditorToolbar. This UI includes all editing capabilities of the control and it can be customized.
+The Telerik UI for .NET MAUI ImageEditor is a control that enables you to visualize and edit images in different file formats in your mobile and desktop application. As an addition to the ImageEditor control you can use the predefined UI with all built-in commands - RadImageEditorToolbar. This UI includes all editing capabilities of the control and it can be customized.
 
 ![.NET MAUI ImageEditor Overview](images/imageeditor-overview.png "ImageEditor Overview")
 
 ## Key features
 
-* Importing and Exporting images&mdash;ImageEditor allows you to import different image formats such as **JPEG, PNG, GIF, BMP** and export images in **JPEG, PNG, GIF, BMP** formats.
+* Importing and Exporting images&mdash;ImageEditor allows you to import different image formats such as `JPEG`, `PNG`, `GIF`, `BMP` and export images in `JPEG`, `PNG`, `GIF`, `BMP` formats.
```

Comment:
Capitalized formats are usually rendered in the default font. Use `monospace` only when describing the extension for the given format, for example, JPEG and `.jpeg`.